apps/sage/ecr/apps/NanoList.tsx

```diff
@@ -0,0 +1,278 @@
+                    "&.Mui-selected": {
+                      // <-- mixing the two classes
+                      backgroundColor: "#ececec",
+                    },
+                    "&.Mui-selected:hover": {
+                      // <-- mixing the two classes
+                      backgroundColor: "#ececec",
+                    },
+                  }}
+                  key={tags.id}
+                  value={tags.id}
+                >
+                  {tags.id}
+                </MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+
+          <Stack
+            direction="row"
+            spacing={2}
+            sx={{ justifyContent: "center", margin: "16px" }}
+          >
+            <Button
+              variant="contained"
+              type="submit"
+              onClick={() => {
+                if (nanoTags.length > 0) handleRegister();
+              }}
+            >
+              Register
+            </Button>
+          </Stack>
+        </Box>
+      </Card>
+
+      <Card className={classes.container}>
+        <Box
+          className={classes.form}
+          sx={{
+            "& .MuiTextField-root": { m: 1, width: "25ch" },
+          }}
+        >
+          <h1>Publish Your Waggle Device</h1>
+          <StepTitle icon="1" label="Enter Nano Information" />
+          <TextField
+            id="uid"
+            label="User ID"
+            variant="outlined"
+            defaultValue={user}
+            className={classes.field}
+          />
+          <FormControl sx={{ m: 1, width: "100%" }}>
+            <InputLabel id="beehive-selection">Beehive</InputLabel>
+            <Select
+              value={beehive}
+              label="Beehive Selection"
+              onChange={handleChange}
+            >
+              <MenuItem value="beehive-dev">Beehive Dev</MenuItem>
+              <MenuItem value="beehive-prod" disabled>Beehive Prod</MenuItem>
+            </Select>
+          </FormControl>
+
+          <TextField
+            id="nano-id"
+            label="Nano ID"
+            variant="outlined"
+            className={classes.field}
+          />
+
+          <Stack
+            direction="row"
+            spacing={2}
+            sx={{ justifyContent: "center", margin: "16px" }}
+          >
+            <Button
+              variant="outlined"
+              color="error"
+              type="submit"
+              onClick={() => console.log("cancel")}
+            >
+              Cancel
+            </Button>
+            <Button
+              variant="contained"
+              type="submit"
+              id="publish-waggle"
+              onClick={handlePublish}
+            >
+              Publish Waggle
+            </Button>
+          </Stack>
+          {regKey ? (
+            <>
+              <h5>Check your download folder for registration keys!</h5>
+            </>
+          ) : (
+            <h5></h5>
+          )}
```

Comment:
This should be simplified to something like:

```
{regKey &&
  <h5>Check your download folder for registration keys!</h5>
}
```

This would be better semantically (and for accessibility) since there wouldn't be an empty header tag.